src/Illuminate/Mail/Transport/MailgunTransport.php

```diff
@@ -93,7 +94,7 @@ public function send(Swift_Mime_SimpleMessage $message, &$failedRecipients = nul
      */
     protected function payload(Swift_Mime_SimpleMessage $message, $to)
     {
-        if (! config('services.mailgun.batch_sending') || count($message->getTo()) === 1) {
+        if (! Config::get('services.mailgun.batch_sending', false) || count($message->getTo()) === 1) {
```

Comment:
Can't call a facade here either. Is the same problem.

Reply:
Maybe I'm not being clear enough. You need to pass batchSending into the constructor, and lookup the config in the service provider.

Reply:
Hi @GrahamCampbell , thanks for your review.

Can I ask why there are 2 occurrences of Config Facade in

1.  src/Illuminate/Auth/Notifications/VerifyEmail.php, line 85
Carbon::now()->addMinutes(Config::get('auth.verification.expire', 60)),
2. src/Illuminate/Database/Console/DumpCommand.php, line 69
 ->withMigrationTable($connection->getTablePrefix().Config::get('database.migrations', 'migrations'))

If we have to inject config in constructor via the provider, are they ok for the components to call facade?

Thanks.